app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
> helpers like the is-array helper would be discovered automatically (because of the dash) in Ember 1.12

I assume that helper registered in initializer will prevent auto-discovery process from happening for `is-array`, is this correct assumption?